### PR TITLE
doc: add release notes about 2023-04-05 release

### PR DESCRIPTION
**Pull request by nikromen**

### CODE REVIEW

File: doc/release-notes/2023-04-05.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+to switch to another search engine since our current search engine
+``whoosh`` seemed to be slow. We managed to speed up the search speed
+by a ton. (`#2534`_)
+
+
+Notable changes
+---------------
+
+- We are checking whether the user can build in the project before
+  uploading SRPM starts. (`#2524`_)
```
Comment (xsuchy) [inline doc/release-notes/2023-04-05.rst:34]: It is better to form it in way "previously ... what was happening ... and now.... how we fixed it".
So this sentence: Previously, when you uploaded src.rpm to be built in non-enabled chroot, Copr emitted an error after the upload of src.rpm. Now we check the chroot before you upload the src.rpm.

File: doc/release-notes/2023-04-05.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+(`#2577`_)
+
+
+Fixing slow searching in Fedora Copr projects
+---------------------------------------------
+
+Searching in Copr was slow and often timeout. We discussed options
+to switch to another search engine since our current search engine
+``whoosh`` seemed to be slow. We managed to speed up the search speed
+by a ton. (`#2534`_)
```
Comment (praiskup) [inline doc/release-notes/2023-04-05.rst:27]: By "a ton" sounds a bit overoptimistic, considering we still have to migrate out rather sooner than later.

File: doc/release-notes/2023-04-05.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+Disable AppStream for existing Copr projects
+--------------------------------------------
+
+We decided to disable AppStream metadata generation for all project
+excluding those mentioned in `Fedora discussion`_ due to very high I/O
+demand of ``appstream-builder`` tool which causes failures in builds
+even for medium-sized projects. If you want to enable generating AppStream
+data for your project, please go to project settings and enable it back.
```
Comment (praiskup) [inline doc/release-notes/2023-04-05.rst:17]: (with a relatively high chance of a random build failure due to a timeout)

**Comment by praiskup** [comment]
Thank you!

File: doc/release-notes/2023-04-05.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+Bugfixes
+--------
+
+`#2519`_ - Fix sorting builds by build time in Web UI
+`#984`_ - Disallow forking too large Copr projects
+`#2538`_ - Don't branch armhfp chroots
+`#2523`_ - Allow cron automation to remove builds
+`#2532`_ - Rubygem rebuild button redirects into an edit page
+`#1171`_ - ``createrepo`` can be run without ``--database`` to save some space
```
Comment (praiskup) [inline doc/release-notes/2023-04-05.rst:52]: This IMO deserves a separate paragraph above.

**Comment by xsuchy** [comment]
ping, can we publish it soon. It is already more than a week from the release.

**Comment by praiskup** [comment]
Announced: https://lists.fedorahosted.org/archives/list/copr-devel@lists.fedorahosted.org/thread/OEP7CC56BU3QYGKUPODPQL4CC4T33KSG/

**Pull request closed without merging**